### ISSUE TITLE
Ionic RC.0 - Tabs selection issue

### ISSUE THREAD

**Issue by anteburazer** [ionic-team/ionic#8610]
#### Short description of the problem:

I'm using ionic tabs and have an issue when trying to select specific tab dynamically.
My application consists of welcome screen, with four buttons, and tabs navigation with four tab pages.

Welcome screen doesn't contain tab navigation and each of four buttons redirects to specific page inside tab menu.
In other words user can select specific tab from the welcome page.

My problem is when selecting a specific tab I always see the first tab page for one moment and after that application goes to selected tab.
So I have a flickering effect at the beginning of transition. I also experienced this in Ionic beta-11 when defining a page template as external html file. Once I included the template inside page component file the transition started working normally.

NOTE: WelcomePage and TabsPage are in the same navigation stack.
#### Expected behavior?

First page shouldn't be visible if user navigates directly to any other page.
#### Plunker that shows an example of your issue

http://plnkr.co/edit/x2ShAmkJWidx3pRrJQAN?p=preview
#### Ionic forum issue

https://forum.ionicframework.com/t/ionic-rc-0-tab-select-issue/66156/3

**System information**

Cordova CLI: 6.3.1
Gulp version:  CLI version 0.2.0
Gulp local:   Local version 3.9.0
Ionic Framework Version: 2.0.0-rc.0
Ionic CLI Version: 2.1.0
Ionic App Lib Version: 2.1.0-beta.1
OS:
Node Version: v6.7.0

**Comment by jgw96** [comment]
Hello, thanks for opening an issue with us. Notes to team: While i do not see a flicker there is a noticeable delay in the above plunker when the tabs page gets navigated too and then the correct tab gets selected.

**Comment by anteburazer** [comment]
Plunker is a little bit slow so you see a delay. In real application, both in browser and mobile devices, this delay is much shorter so it's feeling like a flicker effect.
The first tab is displayed for half a second before going to tab 2.

The interesting thing is that I also experienced this in Ionic beta-11 when defining a page template as external html file. Once I included the template inside .ts component file (inline template) the transition started working normally.
